### ISSUE TITLE
Code question

### ISSUE THREAD

**Issue by tanjia123456** [fursovia/self-adj-dice#5]
Hello, thank you for your contribution.
I want trans the torch version to tensorflow, but I have problem in torch.gather
`
def loss(logits,targets,alpha=1.0,gamma=1.0):
    probs = tf.argmax(logits, 1)
    probs = tf.gather(probs, dim=1, index=targets.unsqueeze(1))(??????)
    probs_with_factor = ((1 - probs) ** alpha) * probs
    loss = 1 - (2 * probs_with_factor + gamma) / (probs_with_factor + 1 +gamma)

    return loss
`
do you mind support tf version? thank you very much
